src/corelibs/U2View/src/ov_msa/MultilineScrollController.cpp

```diff
@@ -534,6 +583,9 @@ void MultilineScrollController::updateChildrenScrollBarsPeivate() {
         if (i == 0) {
             val = hbar->value();
         }
+
+        QSignalBlocker signalBlocker(ui->getUI(i)->getScrollController());
+        Q_UNUSED(signalBlocker);
```

Comment:
1. Q_UNUSED is not needed here
2. getUi(i) can return nullptr.

Reply:
fixed